app/Console/Commands/ImportRockTheVoteReport.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Chompy\Console\Commands;
+
+use Carbon\Carbon;
+use Chompy\ImportType;
+use Illuminate\Console\Command;
+use Chompy\Jobs\ImportFileRecords;
+use Illuminate\Support\Facades\Storage;
+
+class ImportRockTheVoteReport extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rtv:import {reportId}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Import a Rock The Vote Report by Report ID';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $reportId = $this->argument('reportId');
+
+        info('Checking status of report ' . $reportId);
+
+        $report = app('Chompy\Services\RockTheVote')->getReportStatusById($reportId);
+
+        info('Report ' . $reportId . ' status is ' . $report->status);
+
+        if ($report->status === 'complete') {
+            $this->downloadAndImport($reportId, $report->download_url);
+        }
+    }
+
+    /**
+     * Fetches Rock The Vote CSV at given URL for given report ID, and imports it.
+     *
+     * @param int $id
+     * @param string $url
+     */
+    private function downloadAndImport($reportId, $url)
+    {
+        $path = 'uploads/' . ImportType::$rockTheVote . '-report-' . $reportId . '-' . Carbon::now() . '.csv';
+
+        Storage::put($path, app('Chompy\Services\RockTheVote')->getReportByUrl($url));
+
+        info('Downloaded report '.$reportId);
+
+        ImportFileRecords::dispatch(null, $path, ImportType::$rockTheVote, ['report_id' => $reportId])->delay(now()->addSeconds(3));
```

Comment:
Why do we need to delay this by 3 seconds?

Reply:
That's a great question, I honestly don't know. I've been following by example per other jobs in the codebase, that all seem to have it. e.g. https://github.com/DoSomething/chompy/commit/884025eae0fcbab8acb6fbfc1fe7caf28f218a53#diff-e489ef4664b1c24151b9b01896103403

@DFurnes ?

Reply:
oh lol I did it too! I don't remember why though hmmmm

Reply:
Yeah, I can't think of a good reason why we'd want to delay these.

Reply:
Alright, your call then @aaronschachter !

Reply:
I'm going to err on the side of trusting the delays were added for a reason. I wonder if the broadcasting to Pusher could have anything to do with it. Let's saving testing with removing the delays for a separate PR.